README.md

```diff
@@ -79,73 +97,104 @@ Fields that can be included in the response body for `GET /`.
-| --------------------- | -------------------------------------- | ----------------------------------------------- | ---------------------------------------------------------------------------------------- |
-| STAPI - Core          | Core                                   | https://stapi.example.com/v0.1.0/core           | Specifies the STAPI Landing page `/`, communicating conformance and available endpoints. |
-| STAPI - Opportunities | [Opportunities](opportunity/README.md) | https://stapi.example.com/v0.1.0/opportunities  | Enables request of potential tasking opportunities                                       |
-| STAPI - Core          | Core                                   | https://geojson.org/schema/Point.json           | Allows submitting orders with GeoJSON points                                             |
-| STAPI - Core          | Core                                   | https://geojson.org/schema/Linestring.json      | Allows submitting orders with GeoJSON linestrings                                        |
-| STAPI - Core          | Core                                   | https://geojson.org/schema/Polygon.json         | Allows submitting orders with GeoJSON polygons                                           |
-| STAPI - Core          | Core                                   | https://geojson.org/schema/MultiPoint.json      | Allows submitting orders with GeoJSON multi points                                       |
-| STAPI - Core          | Core                                   | https://geojson.org/schema/MultiPolygon.json    | Allows submitting orders with GeoJSON multi polygons                                     |
-| STAPI - Core          | Core                                   | https://geojson.org/schema/MultiLineString.json | Allows submitting orders with GeoJSON multi linestring                                   |
+| **Name**                    | **Specified in**                       | **Conformance URI**                                  | **Description**                                                                          |
+| --------------------------- | -------------------------------------- | ---------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| STAPI - Core                | Core                                   | https://stapi.example.com/v0.1.0/core                | Specifies the STAPI Landing page `/`, communicating conformance and available endpoints. |
+| STAPI - Opportunities       | [Opportunities](opportunity/README.md) | https://stapi.example.com/v0.1.0/opportunities       | Enables request of potential tasking opportunities                                       |
+| STAPI - Async Opportunities | [Opportunities](opportunity/README.md) | https://stapi.example.com/v0.1.0/async-opportunities | Enables async request of potential tasking opportunities                                 |
```

Comment:
I'd prefer this to be `opportunities-async` so it's more apparently related to `opportunities`